artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenwireMessage.java

```diff
@@ -137,6 +137,11 @@ public String getAddress() {
       return null;
    }
 
+   @Override
+   public String getProducerID() {
+      return null;
```

Comment:
This class already existed, its a wip in progress so not my place to remove it. @clebertsuconic may have input

Reply:
Anyone can and should delete unused crap when coming across it, rather than updating it pointlessly. That way it isnt in the way to confuse and/or lead to wasted time for others.

This has been present, seemingly unused from day one, for 6 years now. Theres clearly no actual WIP occurring, but even if there was that is what branches are for.

Reply:
I will remove this separately